packages/addons/addon-depends/moonlight-embedded/package.mk

```diff
@@ -43,7 +41,7 @@ elif [ "$DISPLAYSERVER" = "x11" ]; then
   PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET libvdpau SDL2 ffmpeg libxcb"
 fi
 
-PKG_CMAKE_OPTS_TARGET="$FREESCALE_V4L_INCLUDE"
+PKG_CMAKE_OPTS_TARGET="$FREESCALE_V4L_INCLUDE openssl"
```

Comment:
Are you sure cmake needs openssl parameter? For what purpose?

Reply:
@vpeter4 I have no idea, I build a testfile (moonligeht...v2.zip) without it and it didn't worked according to the testuser

Reply:
Ok, then just leave it. I didn't use this on my tests that's why I'm asking.

Reply:
@vpeter4 reread the post he made and actually misinterpreted it :/ 
I asked if he can clarify.

Reply:
this should be in PKG_DEPENDS_TARGET

Reply:
wasn't needed at all, a misapprehension with the testuser - removed it